flight/Modules/Autotune/autotune.c

```diff
@@ -163,6 +228,8 @@ static void UpdateStabilizationDesired(bool doingIdent) {
 	StabilizationDesiredSet(&stabDesired);
 }
 
+#define MAX_PTS_PER_CYCLE 4
```

Comment:
It inhibits the sleep at the bottom if it doesn't complete the loop, so it'll work at faster rates.  It's there so we spend an appropriate amount of effort looking at manualcontrolcommand since we're in the control critical path.